File: database3.py
# select details from table
import mariadb
import sys
try:
           conn=mariadb.connect(
                      user="root",
                      password="maria",
                      host="127.0.0.1",
                      port =3308,
                      database="dhiraj"
                      )
except mariadb.Error as e:
           print("Error to connect",e)
           sys.exit(1)

print("Database connected successfully :)")
# get cursor
cur=conn.cursor()

try:
    cur.execute("select * from student")
except mariadb.Error as e:
    print("Error",e)

print() # for new line

print("The whole details :)\n")

for id,name,age in cur:
    print("id",id)
    print("name",name)
    print("age",age)
    print() # for new line

print("Thanks for using Python + MariaDb :)")
conn.close()

File: database3.js
//add primary key into the existing table

var mysql=require("mysql")

var conn= mysql.createConnection({host:"localhost",user:"root",password:"rootpassword",database:"node"})

conn.connect(function(err){
    if(err ) throw err;
    console.log("connected :)");

var sql = "alter table nodetable add column id int auto_increment primary key"; //alter the table

conn.query(sql,function(err,result){
    if(err ) throw err;
    console.log("table altered  :)");
});
});

